covid_surveillance/libraries/poolingmethods/source.rb

```diff
@@ -0,0 +1,9 @@
+module PoolingMethods
```

Comment:
This file was deleted in https://github.com/aquariumbio/aq-genome-center/commit/fc748f013d624c0ad2a78da5734865380d2583d2

Reply:
Remove this from the PR please